src/Twig/ContentExtension.php

```diff
@@ -215,4 +216,29 @@ public function getEditLink(Content $content, bool $absolute = false): ?string
             $absolute ? UrlGeneratorInterface::ABSOLUTE_URL : UrlGeneratorInterface::ABSOLUTE_PATH
         );
     }
+
+    public function getTaxonomies(Content $content): Collection
+    {
+        $taxonomies = [];
+        foreach ($content->getTaxonomies() as $t) {
+            $link = $this->urlGenerator->generate('taxonomy', [
+                'taxonomyslug' => $t->getType(),
+                'slug' => $t->getSlug(),
+            ]);
+
+            $taxonomy = [
```

Comment:
use DTO/VO/any other simple model

Reply:
`$t` _is_ already a Taxonomy.. 🤦‍♂️ 

I'll instead add a `link` property to that, and we can just pass in the object to Twig. Saves some lines of code as well.

Reply:
`saving lines of code` is a very poor excuse ;)

Reply:
Poor excuse to do it, but it's a nice side-effect of a change you were making anyways. :-)